src/Arriba/Arriba.Server/Program.cs

```diff
@@ -9,14 +9,34 @@
 using System;
 using System.Diagnostics;
 using AspNetHost = Microsoft.Extensions.Hosting.Host;
+using OpenTelemetry;
+using OpenTelemetry.Trace;
+
 
 namespace Arriba.Server
 {
     internal class Program
     {
+        // Is this considered a static logger? Because it only exists in this context and isn't actually passed into 
+        // other methods
+        private static readonly ActivitySource OpenTelemetryActivitySource = new ActivitySource("Microsoft.Arriba.Server");
+
         private static void Main(string[] args)
         {
             Console.WriteLine("Arriba Local Server\r\n");
+            
+            using var tracerProvider = Sdk.CreateTracerProviderBuilder()
```

Comment:
is tracerProvider being used directly or does it need to be alive?

Reply:
I'm not sure, I think it stays alive.

Here: https://opentelemetry.io/docs/, under the "Configure the OpenTelemetry API" section it says that it acts as the namespace for all the spans under this service

Reply:
I am curious how that namespace state is managed.  some sort of global/static state?